.github/workflows/s3-snapshots.yml

```diff
@@ -19,12 +19,12 @@ jobs:
         with:
           distribution: 'zulu'
           java-version: 12
-      - name: Document stdlib
+      - name: Document coroutines
         uses: gradle/gradle-build-action@v2
         with:
-          arguments: :integration-tests:gradle:integrationTest --tests org.jetbrains.dokka.it.gradle.kotlin.StdlibGradleIntegrationTest --stacktrace
+          arguments: :integration-tests:gradle:integrationTest --tests org.jetbrains.dokka.it.gradle.kotlin.CoroutinesGradleIntegrationTest.kt --stacktrace
         env:
-          DOKKA_IT_AWS_PATH: /home/runner/work/dokka/stdlib
+          DOKKA_PATH_TEST_OUTPUT: /home/runner/work/dokka/coroutines
```

Comment:
somehow `DOKKA_TEST_OUTPUT_PATH` feels more... natural? wdyt?

Reply:
Did it here https://github.com/Kotlin/dokka/pull/2630/commits/9a8a84306f89b9c49eacd2dd4e1978ce5e2f4461